config-editor/config-editor-ui/src/app/services/schema/config-schema-service.ts

```diff
@@ -65,8 +66,8 @@ export class ConfigSchemaService extends SchemaService {
     return config;
   }
 
-  areConfigEqual(config1: any, config2: any): boolean {
-    return areJsonEqual(this.cleanConfig(config1), this.cleanConfig(config2));
+  areConfigEqual(configStore: any, configBeCleaned: any): boolean {
+    return areJsonEqual(configStore, this.cleanConfig(configBeCleaned));
```

Comment:
can we move this to the caller/ component to be clear what is cleaned and what not?